src/server/components/AvslåVedtak.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import { IAvslåVedtak } from '../../typer/dokumentApi';
+import { IAvslåVedtak, avslagÅrsakTilTekst } from '../../typer/dokumentApi';
 
-export const AvslåVedtak: React.FC<IAvslåVedtak> = ({ avslåBegrunnelse }) => {
+export const AvslåVedtak: React.FC<IAvslåVedtak> = ({ avslåÅrsak, avslåBegrunnelse }) => {
```

Comment:
Burde vi unngå verb ? Avslag e.l ?

Reply:
Dette er nok et resultat av to svensker som satt sammen og litt for svak språkgjennomgang fra resten av oss tilbake til 2021 :)